lib/normalize2.js

```diff
@@ -18,19 +16,23 @@ module.exports = function (decl) {
         if (typeof entity === 'string') {
             block = entity;
         } else {
-            block = entity.block;
+            block = entity.block || null;
             elem = entity.elem;
```

Comment:
Well, I guess this is the same for `elem`.

Could you add some cases without block and elem?